### DISCUSSION TITLE
[Help/Question] dokku on fly.io shell webcli / Where To Free-Host Dokku ?

### DISCUSSION THREAD

**Discussion by fire17** [dokku/dokku#5770] [Q&A]
Hi everyone,
I just found out about dokku and it looks really awesome, i'm trying to port over apps i had in heroku and reserve them
I'm lookin for a free testing environmenet, tryed with fly.io but ran into problems
Has anyone managed to freely run dokku on fly.io OR
managed to find a free hosting service that runs dokku ? (even for short experimenting)
Hope someone has some clue.. 
Thanks a lot and have a good one!

**Comment by josegonzalez** [comment]
You can run the docker container locally if you need a testing environment.

**Reply by fire17** [reply]
I guess I can do that and run ngrok on it for exposure...
Is there no one who managed to run this free on fly.io? render? vercel? anything??
